Formula/ucg.rb

```diff
@@ -0,0 +1,33 @@
+class Ucg < Formula
+  desc "grep-like tool for searching large bodies of source code"
+  homepage "https://github.com/gvansickle/ucg"
+  url "https://github.com/gvansickle/ucg/releases/download/0.3.1/universalcodegrep-0.3.1.tar.gz"
+  sha256 "62f9ef88ea5c0777696c4322bed1fb9a3fb62eb85bd053af50f75d42ec259086"
+  head "https://github.com/gvansickle/ucg.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "automake" => :build
+  depends_on "argp-standalone" => :build
+  depends_on "pcre2"
+
+  # superenv's '-march' flag conflicts with build script
+  env :std
```

Comment:
Hi @equal-l2 ,
I believe I've resolved the `-march` issue on upstream master as of 6467465abfbcdddef20a7bde4263e1479feb5264.  It should now build correctly with a `-march` in the CXX or CC env vars.  Can you try master/HEAD and confirm?  I'm also rolling a tarball for release as 0.3.2 which will pick up these changes.